media/js/mozorg/home/takeover-2016.js

```diff
@@ -37,13 +37,31 @@
         });
     }
 
-    // sets the mozorg.takeover state to true indicating that the user has
-    // seen the takeover modal during their current session.
-    function storeSession() {
+    // Check for cookie support (stolen from Modernizr)
+    function testCookies() {
         try {
-            sessionStorage.setItem('mozorg.takeover.seen', 'true');
+            // Create cookie
+            document.cookie = 'cookietest=1';
+            var ret = document.cookie.indexOf('cookietest=') != -1;
+            // Delete cookie
+            document.cookie = 'cookietest=1; expires=Thu, 01-Jan-1970 00:00:01 GMT';
+            return ret;
+        }
+        catch (e) {
+            return false;
+        }
+    }
+
+    // Sets a cookie so you won't see the takeover again for seven days
+    function seenTakover() {
+        var date = new Date();
+        date.setTime(date.getTime() + (7 * 24 * 60 * 60 * 1000));
+        var expires = date.toUTCString();
+
+        try {
```

Comment:
Now we're using cookies and a feature detect, there should be no need for this `try/catch`, right?